pkg/v1/providers/config_default.yaml

```diff
@@ -596,15 +596,35 @@ AVI_CONTROL_PLANE_HA_PROVIDER: false
 #! Antrea CNI configuration
 #! ---------------------------------------------------------------------
 ANTREA_NO_SNAT: false
+ANTREA_DISABLE_UDP_TUNNEL_OFFLOAD: false
 ANTREA_TRAFFIC_ENCAP_MODE: "encap"
+ANTREA_EGRESS_EXCEPT_CIDRS: ""
+ANTREA_PORT_LOCAL_ENABLED: true
+ANTREA_PORT_LOCAL_PORTRANGE: 61000-62000
+ANTREA_PROXY_ALL: true
+ANTREA_PROXY_NODE_PORT_ADDR: ""
```

Comment:
Maybe add an "S" after ADDR, and use NODEPORT instead of NODE_PORT - ANTREA_PROXY_NODEPORT_ADDRS